ext/BraketSimulatorBraketExt/BraketSimulatorBraketExt.jl

```diff
@@ -0,0 +1,166 @@
+                                    (:(Braket.Y), :(BraketSimulator.Y)),
+                                    (:(Braket.Z), :(BraketSimulator.Z)),
+                                    (:(Braket.I), :(BraketSimulator.I)),
+                                    (:(Braket.H), :(BraketSimulator.H)),
+                                    (:(Braket.V), :(BraketSimulator.V)),
+                                    (:(Braket.Vi), :(BraketSimulator.Vi)),
+                                    (:(Braket.S), :(BraketSimulator.S)),
+                                    (:(Braket.Si), :(BraketSimulator.Si)),
+                                    (:(Braket.T), :(BraketSimulator.T)),
+                                    (:(Braket.Ti), :(BraketSimulator.Ti)),
+                                    (:(Braket.ECR), :(BraketSimulator.ECR)),
+                                    (:(Braket.CNot), :(BraketSimulator.CNot)),
+                                    (:(Braket.CY), :(BraketSimulator.CY)),
+                                    (:(Braket.CZ), :(BraketSimulator.CZ)),
+                                    (:(Braket.CV), :(BraketSimulator.CV)),
+                                    (:(Braket.CCNot), :(BraketSimulator.CCNot)),
+                                    (:(Braket.Swap), :(BraketSimulator.Swap)),
+                                    (:(Braket.ISwap), :(BraketSimulator.ISwap)),
+                                    (:(Braket.CSwap), :(BraketSimulator.CSwap)),
+                                   )
+    @eval Base.convert(::Type{BraketSimulator.Operator}, g::$braket_sim) = $simulator_sym()
+    @eval Base.convert(::Type{Braket.Operator}, g::$simulator_sym) = $braket_sim()
+end
+
+for (braket_sim, simulator_sym) in ((:(Braket.Rx), :(BraketSimulator.Rx)),
+                                    (:(Braket.Ry), :(BraketSimulator.Ry)),
+                                    (:(Braket.Rz), :(BraketSimulator.Rz)),
+                                    (:(Braket.GPi), :(BraketSimulator.GPi)),
+                                    (:(Braket.GPi2), :(BraketSimulator.GPi2)),
+                                    (:(Braket.XX), :(BraketSimulator.XX)),
+                                    (:(Braket.XY), :(BraketSimulator.XY)),
+                                    (:(Braket.YY), :(BraketSimulator.YY)),
+                                    (:(Braket.ZZ), :(BraketSimulator.ZZ)),
+                                    (:(Braket.PSwap), :(BraketSimulator.PSwap)),
+                                    (:(Braket.PhaseShift), :(BraketSimulator.PhaseShift)),
+                                    (:(Braket.CPhaseShift), :(BraketSimulator.CPhaseShift)),
+                                    (:(Braket.CPhaseShift00), :(BraketSimulator.CPhaseShift00)),
+                                    (:(Braket.CPhaseShift01), :(BraketSimulator.CPhaseShift01)),
+                                    (:(Braket.CPhaseShift10), :(BraketSimulator.CPhaseShift10)),
+                                   )
```

Comment:
any reason that MS and PRx are not in this list?

Reply:
Because they both have more than one angle argument

Reply:
They're both present further down in the file :)

Reply:
Oh, right, sorry. I guess I stopped reading when I got to the noise operators, didn't see them down there. 😦 Thanks!